series/_posts/2010-10-7-cycle.md

```diff
@@ -0,0 +1,63 @@
+  
+  var parseDate = d3.time.format('%Y%m%d').parse;
+  data.forEach(function(d) {
+      d.date = parseDate(d.date);
+  });
+
+  var dayScale = d3.scale.linear()
+      .domain([-0.5, 6.5])
+      .range([0, width]);
+
+  var salesScale = d3.scale.linear()
+      .domain(fc.utilities.extent(data, 'sales'))
+      .range([height, 0])
+      .nice();
+
+  var subScale = d3.scale.linear()
+      .domain(fc.utilities.extent(data, 'date'));
+
+  var subAxis = fc.series.axis()
+      .tickSize(0)
+      .ticks(0)
+      .baseline(function(d) {
+          return d3.mean(d, function(d) { return d.sales; });
+      });
+
+  var lineSeries = fc.series.line()
+      .xValue(function(d) { return d.date; })
+      .yValue(function(d) { return d.sales; });
+
+  var multiSeries = fc.series.multi()
+          .series([subAxis, lineSeries]);
+
+  var cycle = fc.series.cycle()
+      .barWidth(fc.utilities.fractionalBarWidth(0.9))
+      .yScale(salesScale)
+      .xScale(dayScale)
+      .xValue(function(d) { return d.date.getDay(); })
+      .subScale(subScale)
+      .subSeries(multiSeries);
+
+  container.append('g')
+      .datum(data)
+      .call(cycle);
+---
+
+The cycle series component allows the creation of cycle plots which can help identify cyclical patterns nested in time series data. The following example shows some arbitrary sales data grouped by weekday:
+
```

Comment:
I think this example could do with a little more explanation, the use of two different scales is notable! The `xScale` is the 'outer' scale, with the cycle series `xValue` mapping the data onto this scale, while the `subScale` is the inner scale, with the `xValue` of the line series corresponding to this scale.

I'd also mention the axis, that it is the same as the d3 axis, but with the addition of a baseline function which maps the array of data to a y-location (in this example)